toolchain/scripts/toolchain.sh

```diff
@@ -551,6 +553,18 @@ function generate_oasisctl() {
   log "[generate_oasisctl] Done"
 }
 
+function download_oasisctl() {
+  oasisctlVersion=$(curl -I https://github.com/arangodb-managed/oasisctl/releases/latest | awk -F '/' '/^location/ {print  substr($NF, 1, length($NF)-1)}')
+  log "[download_oasisctl] Downloading oasisctl version $oasisctlVersion"
+  cd /tmp
+  rm -r oasisctl.zip bin
```

Comment:
When the generation runs for 3.10, 3.11, and 3.12, then for the first version (3.10) the files do not exist. In the 3.11 and 3.12 runs, they do exist. Based on this observation, can't we skip the download of the same oasisctl version for 3.11 and 3.12 simply by checking if we already have the binary?

Reply:
Solved in Commit https://github.com/arangodb/docs-hugo/pull/288/commits/319810ac42f8472774abe4146492143c6f6afd3e

oasisctl is downloaded only if /tmp/oasisctl.zip is not present already and once for the entire toolchain run
https://app.circleci.com/pipelines/github/arangodb/docs-hugo/2750/workflows/148bb9f5-9ef7-41ee-a606-3f43a2a6cd82/jobs/7010